txpool/txpool.go

```diff
@@ -467,6 +468,11 @@ func (p *TxPool) processEvent(event *blockchain.Event) {
 // validateTx ensures the transaction conforms to specific
 // constraints before entering the pool.
 func (p *TxPool) validateTx(tx *types.Transaction) error {
+	// FIXME: Accepts only legacy transaction only for now
```

Comment:
Leftover?

Reply:
This is not leftover right now. But we need to fix when we introduce other typed transaction. We can keep it right now

Reply:
@zivkovicmilos

Reply:
Resolving the nitpick so we can merge this out